src/main/java/harmonised/pmmo/api/events/EnchantEvent.java

```diff
@@ -6,18 +6,18 @@
 import net.minecraftforge.event.entity.player.PlayerEvent;
 
 public class EnchantEvent extends PlayerEvent{
-	EnchantmentInstance enchant;
 	ItemStack stack;
+	int levelsSpent;
 	
-	public EnchantEvent(Player player, ItemStack stack, EnchantmentInstance enchantment) {
```

Comment:
This change not only changes the core functionality of the event, but reduces the information provided to consumers.  It is important to note that this is an event class that is meant to be posted by mods when their enchanting behavior is performed such that pmmo can then listen for this event.  Another mod may not use levels spent in their custom enchanting behavior, which not only breaks their compat, but adds nothing of value to pmmo.

Reply:
Understood. I wasn't aware that other projects might have dependencies to this class. This invalidates the whole idea